multiqc/modules/rseqc/bam_stat.py

```diff
@@ -51,15 +52,20 @@ def parse_reports(self):
                 d['unique_percent'] = (float(d['mapq_gte_mapq_cut_unique']) / t)*100.0
             if 'reads_mapped_in_proper_pairs' in d:
                 d['proper_pairs_percent'] = (float(d['reads_mapped_in_proper_pairs']) / t)*100.0
-        
+
         if len(d) > 0:
             if f['s_name'] in self.bam_stat_data:
                 log.debug("Duplicate sample name found! Overwriting: {}".format(f['s_name']))
             self.add_data_source(f, section='bam_stat')
+            #Check if SE or PE
+            
+            if d['read_1'] and d['read_2'] and d['reads_mapped_in_proper_pairs'] and d['proper-paired_reads_map_to_different_chrom'] != 0:
```

Comment:
And in fact, thinking more about it - we probably only need to check the value of `read_2`... I think that should always be `> 0` for paired end data.